packages/forma-36-react-components/package.json

```diff
@@ -114,8 +114,8 @@
     "truncate": "^2.0.1"
   },
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "^16.8.6 || ^17.0.0",
+    "react-dom": "^16.8.6 || ^17.0.0"
```

Comment:
I think this could just be `">= 16.8"` no?

Reply:
I think the range is more future proof, but there's not a big difference either way in my opinion.